README.md

```diff
@@ -201,7 +201,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [json5](https://github.com/Joakker/tree-sitter-json5) (maintained by @Joakker)
 - [x] [JSON with comments](https://gitlab.com/WhyNotHugo/tree-sitter-jsonc.git) (maintained by @WhyNotHugo)
 - [x] [julia](https://github.com/tree-sitter/tree-sitter-julia) (maintained by @mroavi, @theHamsta)
-- [x] [kotlin](https://github.com/Joakker/tree-sitter-kotlin) (maintained by @Joakker)
+- [x] [kotlin](https://github.com/fwcd/tree-sitter-kotlin) (maintained by @fwcd)
```

Comment:
looks good to me to use the parser that was also used in Kotlin language server. I'd like to hear the opinion of @Joakker

Reply:
Sorry I haven't been on. Life's been rough lately. If it's the same parser as the kotlin langserver, then it's fine by me